metadata-comparison/src/main/java/org/codice/nitf/metadatacomparison/FileComparer.java

```diff
@@ -97,15 +103,14 @@ private void generateOurMetadata() {
     }
 
     private void outputData() {
-        try {
-            FileWriter fstream = new FileWriter(filename + OUR_OUTPUT_EXTENSION);
+        try (FileWriter fstream = new FileWriter(filename + OUR_OUTPUT_EXTENSION)) {
             out = new BufferedWriter(fstream);
             out.write("Driver: NITF/National Imagery Transmission Format\n");
             out.write("Files: " + filename + "\n");
             if (segment1 == null) {
-                out.write(String.format("Size is 1, 1\n"));
+                out.write(String.format("Size is 1, 1%n"));
```

Comment:
I'm slightly concerned about this, because the goal of this code is to match the GDAL (`gdalinfo`) output, and I think it uses newlines on windows. Its not a big deal though - easy cleanup later if it turns out to be a problem.

Reply:
`%n` should still resolve to a new line according to the formatter syntax. By changing it to the `%` it should be a platform-specific line separator. I made this change and tested on a windows dev environment, hopefully it doesn't introduce any issues 🤞